smarts/sstudio/generators.py

```diff
@@ -236,6 +236,21 @@ def _writexml(
                     **actor.junction_model,
                 )
 
+            if traffic.trips is not None:
+                for actor in {trip.actor for trip in traffic.trips}:
```

Comment:
I think what we should do is combine the trip actors and flow actors to ensure they are unique.

```
actors_for_vtype = {
   actor for flow in traffic.flows for actor in flow.actors.keys()
}
if traffic.trips:
    actors_for_vtype |= {trip.actor for trip in traffic.trips}

for actor in actors_for_vtype:
    # Generate vType
    ...
```

Reply:
@AisenGinn Thanks.